notebooks/ISS_Pipeline_-_Breast_-_1_FOV.ipynb

```diff
@@ -158,8 +158,8 @@
     "from starfish.pipeline.filter import Filter\n",
     "\n",
     "# filter raw data\n",
-    "disk_size = 15  # disk as in circle\n",
-    "filt = Filter.WhiteTophat(disk_size, verbose=True)\n",
+    "masking_radius = 15  # disk as in circle\n",
```

Comment:
circle not 3d!

Reply:
This looks like it is only used in watershed, which is still fundamentally 2d, so that is OK for now. I haven't done enough reading to know if we can/will want to adapt that method.